src/pages/workspaces/workspace/notebooks/NotebookLauncher.js

```diff
@@ -101,9 +101,12 @@ const NotebookLauncher = _.flow(
 
     return h(Fragment, [
       (Utils.canWrite(accessLevel) && canCompute && !!mode && clusterStatus === 'Running') ?
-        h(NotebookEditorFrame, { key: cluster.clusterName, workspace, cluster, notebookName, mode }) :
         h(Fragment, [
-          h(PreviewHeader, { queryParams, cluster, refreshClusters, notebookName, workspace, readOnlyAccess: !(Utils.canWrite(accessLevel) && canCompute) }),
+          cluster.labels.welderInstallFailed ? h(PlaygroundHeader, {}) : null,
```

Comment:
nit: since this is an empty props object, it can just be omitted. Also, this could use `&&` for conditional rendering.